docs/content/reference/migration-0-9.md

```diff
@@ -3,4 +3,238 @@ title: Migration to 0.9
+Notes:
+ - `class_descriptions` has become `context`
+ - `rr.ClassDescription` now requires `info` to be provided rather than defaulting to 0.
+ - `rr.AnnotationInfo` now requires `id` to be provided rather than defaulting to 0.
+
+### `log_arrow`
+Replace with [Arrows3D](data_types/archetypes/arrows3d.md)
+
+Python docs: [Arrows3D.__init__](https://ref.rerun.io/docs/python/HEAD/common/spatial_archetypes/#rerun.Arrows3D.__init__)
+
+Notes:
+ - `with_scale` has become `radii`, which entails dividing by 2 as necessary.
+ - `identifiers` has become `instance_keys`.
+
+### `log_cleared`
+Replace with [Clear](data_types/archetypes/clear.md)
+
+Python docs: [Clear.__init__](https://ref.rerun.io/docs/python/HEAD/common/clearing_entities/#rerun.Clear.__init__)
+
+### `log_depth_image`
+Replace with [DepthImage](data_types/archetypes/depth_image.md)
+
+Python docs: [DepthImage.__init__](https://ref.rerun.io/docs/python/HEAD/common/images/#rerun.DepthImage.__init__)
+
+Notes:
+ * `image` has become `data`
+
+### `log_disconnected_space`
+Replace with [DisconnectedSpace](data_types/archetypes/disconnected_space.md)
+
+Python docs: [DisconnectedSpace.__init__](https://ref.rerun.io/docs/python/HEAD/common/transforms_and_coordinate_systems/#rerun.DisconnectedSpace.__init__)
+
+### `log_extension_components`
+Replace with `AnyValues`
+
+Python docs: TODO(jleibs): Link pending
+
+Notes:
+ - Instead of passing `ext` as a dictionary, `AnyValues` now maps all keyword arguments directly to components.
+   - `rr.log_extension_components(..., ext={'mydata': 1})` becomes `rr.log(... rr.AnyValues(mydata=1))`
+
+### `log_image`
+Replace with [Image](data_types/archetypes/image.md)
+
+Python docs: [Image.__init__](https://ref.rerun.io/docs/python/HEAD/common/images/#rerun.Image.__init__)
+
+Notes:
+ * `image` has become `data`
+ * `jpeg_quality` is now handled by calling `.compress(jpeg_quality=...)` on the image after constructing it.
+
+### `log_image_file`
+Replace with `ImageEncoded`
+
+Python docs: [ImageEncoded.__init__](https://ref.rerun.io/docs/python/HEAD/common/images/#rerun.ImageEncoded.__init__)
+
+Notes:
+ - `img_bytes` and `img_path`
+
+
+### `log_line_strip`, `log_line_strips_2d`, `log_line_strips_3d`, `log_line_segments`
+Replace with [LineStrips2D](data_types/archetypes/line_strips2d.md) or [LineStrips3D](data_types/archetypes/line_strips3d.md)
+
+Python docs: [LineStrips2D.__init__](https://ref.rerun.io/docs/python/HEAD/common/spatial_archetypes/#rerun.LineStrips2D.__init__), [LineStrips3D.__init__](https://ref.rerun.io/docs/python/HEAD/common/spatial_archetypes/#rerun.LineStrips3D.__init__)
+
+Notes:
+ - `log_line_segments` used to take an array of shape (2 * num_segments, 2 or 3) (where points were connected in
+even-odd pairs). Instead this is now handled by a batch of `LineStrips` all of length 2. Note that `LineStrips` now
+takes an array of shape (num_strips, num_points_per_strip, 2 or 3). You can use convert to the new format using the snippets:
```

Comment:
This makes it sounds like the array needs to be rectangular, i.e. that all strips in the same batch has the same number of points in it.

Rather, it takes a list of arrays, each of shape `(num_points_in_strip, 2 or 3)`

Reply:
Good catch.